README.md

```diff
@@ -123,6 +130,43 @@ the official Rust compiler:
 
 [Boehm Demers Weiser GC (BDWGC)]: https://github.com/ivmai/bdwgc
 
+### The `Gc<T>` smart pointer
+
+Alloy provides a new smart pointer type, `Gc<T>`, which allows for shared
+ownership of a value of type `T` allocated in the heap and managed by a garbage
+collector. 
+
+```rust
+use std::gc::Gc;
+
+fn main() {
+    let a = Gc::new(123);
+}
+```
+
+This creates a garbage collected object containing the `u64` value `123`. A
+`Gc`'s data is stored in a `GcBox` internally (similar to `Rc`'s
+implementation).
+
+### Interior mutability
+
+There is no way to mutate, or obtain a mutable reference (`&mut T`) to a `Gc<T>`
```

Comment:
s/to a/to the contents of a/ ?

Reply:
Fixed here: 01bf7083554ca91768c138e4b4fe66d9448a3d13